webots_ros2_foxy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# inherit both the ROS2 and Webots containers
+ARG BASE_IMAGE_WEBOTS=cyberbotics/webots:R2021a-ubuntu20.04
+ARG IMAGE_ROS2=niurover/ros2_foxy:latest
+
+FROM $BASE_IMAGE_WEBOTS AS base
+FROM $IMAGE_ROS2 AS image_ros2
+
+# resolve a missing dependency for webots demo
+RUN sudo apt update && apt install -y libxtst6
```

Comment:
Generally speaking, all of your package installs should take the following format:

```
RUN apt-get update && apt-get install -y \
		<package-names, 1 per line> \
		&& apt-get clean \
		&& rm -rf /var/lib/apt/lists/*
```

No need for sudo, because everthing run with the RUN command is done as root, and the last two lines mean we aren't including unneccessary apt lists in the final image, which inflate the image size dramatically.

Reply:
Noted, thanks. I'll make these changes soon.

Reply:
I fixed the format, rebuilt the container, pushed it to the `niurover/webots_ros2_foxy` docker repository, and tested it to make sure Webots still works.